oxd-server/src/main/java/org/gluu/oxd/server/service/RpSyncService.java

```diff
@@ -48,6 +48,14 @@ public Rp getRp(String oxdId) {
 
         return sync(rp);
     }
+    //this method added to skip the vadidations while running test cases.
+    public Rp getRpTest(String oxdId) {
```

Comment:
It seems you want to get `sync` method. Then make is public and test it. This method looks redundant.